### PR TITLE
tickets/DM-46309: Add open and close mirror covers SAL scripts for MT

### PR DESCRIPTION
**Pull request by cvillalon**

### CODE REVIEW

File: python/lsst/ts/standardscripts/maintel/close_mirror_covers.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    async def configure(self, config):
+        if self.mtcs is None:
+            self.mtcs = MTCS(
+                domain=self.domain, intended_usage=MTCSUsages.All, log=self.log
+            )
+            await self.mtcs.start_task
+
+    def set_metadata(self, metadata):
+        metadata.duration = self.mirror_covers_time_guess
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/maintel/close_mirror_covers.py:63]: If you add the cover timeout as I suggested in the other ticket to `MTCS` you could use that one here instead.

File: python/lsst/ts/standardscripts/maintel/close_mirror_covers.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    async def run(self):
+        await self.mtcs.assert_all_enabled()
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/maintel/close_mirror_covers.py:66]: Can you please add a checkpoint here?

```
await self.checkpoint("Closing mirror covers.")
```

File: python/lsst/ts/standardscripts/maintel/open_mirror_covers.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+    async def configure(self, config):
+        if self.mtcs is None:
+            self.mtcs = MTCS(
+                domain=self.domain, intended_usage=MTCSUsages.All, log=self.log
+            )
+            await self.mtcs.start_task
+
+    def set_metadata(self, metadata):
+        metadata.duration = self.mirror_covers_time_guess
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/maintel/open_mirror_covers.py:63]: Same comment as above about the timeout.

File: python/lsst/ts/standardscripts/maintel/open_mirror_covers.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+    async def run(self):
+        await self.mtcs.assert_all_enabled()
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/maintel/open_mirror_covers.py:66]: Can you please add a `checkpoint` here?

```
await self.checkpoint("Opening mirror covers.")
```

File: tests/test_maintel_close_mirror_covers.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+import logging
+import random
+import unittest
+
+from lsst.ts import standardscripts
+from lsst.ts.standardscripts.maintel import CloseMirrorCovers
+
+random.seed(47)  # for set_random_lsst_dds_partition_prefix
+
+logging.basicConfig()
```
Comment (tribeiro) [inline tests/test_maintel_close_mirror_covers.py:32]: you can remove this.

File: tests/test_maintel_open_mirror_covers.py [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+import logging
+import random
+import unittest
+
+from lsst.ts import standardscripts
+from lsst.ts.standardscripts.maintel import OpenMirrorCovers
+
+random.seed(47)  # for set_random_lsst_dds_partition_prefix
+
+logging.basicConfig()
```
Comment (tribeiro) [inline tests/test_maintel_open_mirror_covers.py:32]: you can remove this.

**Review by tribeiro — APPROVED** [review]
Looks good, I do have a couple inline comments I hope you will consider before merging.

File: python/lsst/ts/standardscripts/maintel/close_mirror_covers.py (hunk 1 above)
Comment (cvillalon) [reply python/lsst/ts/standardscripts/maintel/close_mirror_covers.py]: Done with a custom timeout

**Pull request merged**